tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/Handlers/CheckRunHandler.cs

```diff
@@ -14,31 +14,6 @@ namespace Azure.Sdk.Tools.CheckEnforcer.Handlers
 {
     public class CheckRunHandler : Handler<CheckRunEventPayload>
     {
```

Comment:
@mitchdenny, what are ramifications of removing these EventIds that the logger was using?

Reply:
Nothing in practice. It was just an idea that didn't pan out.